styles/scss/components/closed-caption.scss

```diff
@@ -0,0 +1,13 @@
+.video-react {
+  .video-react-closed-caption {
+    cursor: pointer;
+    @include flex(none);
+    @extend .video-react-icon;
+    @extend .video-react-icon-closed-caption;
+  }
+
+  video::-webkit-media-text-track-container {
+    -webkit-transform: translateY(-30px) !important;
```

Comment:
Is it a must to have `!important` here? I think we should avoid it so that it's can be overwritten if users need to

Reply:
you are right, forgot to remove it.

Reply:
Removed